src/css/advertisement.css

```diff
@@ -0,0 +1,120 @@
+/* CLASS FOR BACKGROUND IMAGE THE SECTION */
+
```

Comment:
@media screen and (min-width: 1280px) {
.advertisement {margin-bottom: 100px;}}

@media screen and (min-width: 768px) {
.advertisement {margin-bottom: 80px;}}

.advertisement {margin-bottom: 60px;}